css/danti.css

```diff
@@ -0,0 +1,306 @@
+.contener{
+    display: grid;
+    height: 100vh;
+    grid-template-columns:0.5fr 1fr 1fr 1fr 1fr 0.5fr;
+    grid-template-rows: 0.5fr 3fr 3fr 0.5fr ;
+    grid-template-areas: 
+        "nav nav nav nav nav nav"
+        ". form form img img img"
+        ". form form img img img"
+        ". hr hr hr hr hr"
+        ". foot foot foot foot foot"
+    ;
+
+}
+
+.nav_all{
```

Comment:
One of the most powerful parts of CSS is "re-usability": even though this is a totally separate page to the homepage, you don't need to write the header/navigation styles all over again. You can just include the main `style.css` file in a `<link>` tag in your store page as well as your new store CSS file. This means as long as you use the same HTML for the header and navigation, then they will look identical too (with one exception - in the design for the store page the "Store" link is always highlighted in orange, to show which page you're on).